selfdrive/car/__init__.py

```diff
@@ -245,12 +245,21 @@ def update(self, current_value, current_counter):
 
 CarInfos = Union[CarInfo, List[CarInfo]]
 
+
+@dataclass
+class CarSpecs:
```

Comment:
any reason to make this its own struct?

Reply:
https://github.com/commaai/openpilot/pull/31550#discussion_r1500130053

Reply:
hmm, it's not super obvious that's happening. might be better to define them as constants above

Reply:
I agree, it should be easy to group multiple platforms together with the same specs, that's one thing the old method made easy, so it should be as obvious and easy to do the same with the refactor. I'm not sure what that looks like yet though